flink-dist/src/main/flink-bin/conf/logback-yarn.xml

```diff
@@ -17,7 +17,7 @@
   -->
 
 <configuration>
-    <appender name="file" class="ch.qos.logback.core.FileAppender">
+    <appender name="file" class="ch.qos.logback.core.rolling.RollingFileAppender">
```

Comment:
add a `rollingPolicy` to limit the size of log files.

Reply:
Will add soon.